services/whatpulse/whatpulse.tester.js

```diff
@@ -0,0 +1,22 @@
+import Joi from 'joi'
+import { createServiceTester } from '../tester.js'
+import { isOrdinalNumber } from '../test-validators.js'
+export const t = await createServiceTester()
+
+t.create('WhatPulse user as user id, metric not from Ranks')
+  .get('/uptime/user/179734.json')
+  .expectBadge({ label: 'uptime', message: Joi.string() })
```

Comment:
`Joi.string()` will match anything, including 'not found', 'inaccessible', etc so we're not really checking anything useful here. Once we've implemented some formatting, lets implement some more meaningful checks under test.

Reply:
modified